modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -13,21 +13,45 @@ You can also grant developers and other users different permissions:
 * To monitor user-defined projects.
 * To configure the components that monitor user-defined projects.
 * To configure alert routing for user-defined projects.
+* To manage alerts and silences for user-defined projects.
```

Comment:
I know this is outside the scope of this PR, but IBM style guide suggests something like:

```
You can also grant developers and other users and of the following permissions:
* Monitoring user-defined projects
* Configuring the components that monitor user-defined projects
* Configuring alert routing for user-defined projects
* Managing alerts and silences for user-defined projects
```